pkg/metrics/exporters/prometheus/prometheus_exporter.go

```diff
@@ -15,6 +16,7 @@ import (
 const (
 	Name      = "prometheus"
 	namespace = "gatekeeper"
+	readHeaderTimeout   = 60 * time.Second
```

Comment:
this is due to `pkg/metrics/exporters/prometheus/prometheus_exporter.go:69:17: G112: Potential Slowloris Attack because ReadHeaderTimeout is not configured in the http.Server (gosec)`

Looks like nginx sets it for 60s. Let me know if there is a better value here.
http://nginx.org/en/docs/http/ngx_http_core_module.html#client_header_timeout